apps/guardian-ui/src/GuardianApi.ts

```diff
@@ -191,12 +200,22 @@ enum AdminRpc {
   fetchEpochCount = 'fetch_epoch_count',
   consensusStatus = 'consensus_status',
   connectionCode = 'connection_code',
+  config = 'config',
+  module = 'module',
+}
+
+export enum LightningModuleRpc {
+  listGateways = 'list_gateways',
```

Comment:
Should we migrate this to shared types as well? Would these RPCs calls be part of gateway UI?

Reply:
These are only in Guardian UI. Afaik